guide-us-english.tex

```diff
@@ -257,7 +257,7 @@ \subsection{Scaling the SoS}
 
 \subsection{Scaling the Product Owner Team}
 
-Just as SoS's can grow into SoSoS's, Product Owner Teams also expand by the same mechanism. There is no specific term associated with these expanded units, nor do the CPO's of them have specific expanded titles. We encourage each organization to develop their own. For the following diagrams, we have chosen to add an additional ``Chief'' to the title of those PO's as they magnify out.
+Just as SoS's can grow into SoSoS, Product Owner Teams also expand by the same mechanism. There is no specific term associated with these expanded units, nor do the CPO's of them have specific expanded titles. We encourage each organization to develop their own. For the following diagrams, we have chosen to add an additional ``Chief'' to the title of those PO's as they magnify out.
```

Comment:
This is not consistent with an earlier commit where SoSs was changed to SoS. Then also SoS's should be changed to SoS.